content/040_observability_monitoring_lab/42_lab1.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "1. Find the Root Cause"
+chapter: true
+weight: 8
+---
+
+## Find the Root Cause
+
+The TravelLogic app that you support faces unusual latency in loading the app web page, also sometimes resulting in outage in the us-area-1. You need to find out the root cause of the latency and outages ASAP.
```

Comment:
us-area-1 might be difficult for audience to understand. Can we say us-east-1? ASAP - as soon as possible.